platform/darwin/src/MGLStyle.mm

```diff
@@ -14,6 +14,8 @@
 #import "MGLStyle_Private.h"
 #import "MGLStyleLayer_Private.h"
 #import "MGLSource_Private.h"
+#import "MGLLight.h"
```

Comment:
Nit: This import statement is redundant to the MGLLight_Private.h import statement below.